### PR TITLE
Plugin general fixes ebi

### PR DESCRIPTION
**Pull request by antgonza**
This fixes all EBI tests. Built on top of https://github.com/biocore/qiita/pull/1540

### CODE REVIEW

File: qiita_ware/ebi.py [hunk 1]
```diff
@@ -161,7 +161,7 @@ def __init__(self, artifact_id, action):
         self.experiment_xml_fp = None
         self.run_xml_fp = None
         self.submission_xml_fp = None
-        self.pmids = self.study.pmids
+        self.publications = self.study.publications
```
Comment (mortonjt) [inline qiita_ware/ebi.py]: Redundant?

Comment (antgonza) [inline qiita_ware/ebi.py]: This is a good point. It's not really redundant as self.publications will be a list of the publications and self.study.publications is a query to the DB to retrieve the publications. I could remove but I like the idea that in the init we make sure that everything that is possible looks sane.

**Comment by josenavas** [comment]
:+1:

File: qiita_ware/ebi.py (hunk 1 above)
Comment (mortonjt) [inline qiita_ware/ebi.py]: Ok. That's fine.  Just wanted to make sure thats on the radar.

**Pull request merged**